app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user,   only: [:edit, :update]
+  before_action :admin_user,     only: :destroy
+
+  def new
```

Comment:
Your views/users/new.html.erb seems to be empty. I am getting this when I try to sign up:

![image](https://user-images.githubusercontent.com/6015315/57563839-81895480-73d5-11e9-8010-e251f2fbeaf0.png)

Your views/users/new.html.erb:

![image](https://user-images.githubusercontent.com/6015315/57563847-9e258c80-73d5-11e9-9156-70ba75527df7.png)

Reply:
Signup form is not need in this project at all.. But ok...